corehq/ex-submodules/casexml/apps/phone/restore_caching.py

```diff
@@ -24,7 +23,6 @@ def _restore_payload_path_cache_key(domain, user_id, sync_log_id, device_id):
         domain=domain,
         prefix=RESTORE_CACHE_KEY_PREFIX,
         user_id=user_id,
-        version='2.0',
```

Comment:
Yeah, so the reasoning is, it looked to me like it wasn't being done consistently anyway, so having it look like it was working isn't as good as just not having it there at all; as to the option to fix it, V1 is very, very deprecated, so there's really only one version, 2.0. I'm not too worried about the use case of someone creating a new app on their old domain from 2012 and installing that on the phone of one of their users who recently restored.